src/textacy/constants.py

```diff
@@ -21,6 +21,40 @@
 OBJ_DEPS: set[str] = {"attr", "dobj", "dative", "oprd"}
 AUX_DEPS: set[str] = {"aux", "auxpass", "neg"}
 
+"""
+Ordinal points of the token.is_quote characters, matched up by start and end.
+
+source:
+switch = "\"\'"
+start = "“‘```“‘«‹「『„‚"
+end = "”’’’’”’»›」』”’"
+
+"""
```

Comment:
nit: could you move this docstring to directly below (rather than above) the object's assignment?